### PR TITLE
Feature/last mile before press

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: src/pages/index.md [hunk 1]
```diff
@@ -29,11 +29,12 @@ peril_forest:
   bottom_paragraph: Läs allt om skyddet i appen.
   heading: 'Hedvig innehåller allt en hemförsäkring brukar innehålla, plus drulle'
 customerSources:
-  headline: Redan försäkrad? Vi sköter bytet
+  headline: "Redan försäkrad? Vi sköter\_bytet"
```
[the remark this answers is not in the document]
Comment (hedvigoscar) [reply src/pages/index.md]: Yes! `\_` maps to a nbsp, preventing a single word (bytet) being on its own row (widow and orphan)

File: static/admin/config.yml [hunk 2]
```diff
@@ -575,9 +578,35 @@ collections:
               - label: Bottom paragraph
                 name: bottom_paragraph
                 widget: string
-          - label: Bottom CTA
-            name: bottom_cta
-            widget: string
+          - label: Customer source section
```
[the remark this answers is not in the document]
Comment (hedvigoscar) [reply static/admin/config.yml]: Yes 😭 

This is one of the weaker points about our CMS setup right now: due to netlify CMS having poor multi-language authoring the current setup is just to straight duplicate the CMS config for both languages. 🤢

File: src/pages/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (palmfjord) [reply src/pages/index.md]: Wo0t. Interesting!

File: static/admin/config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (palmfjord) [reply static/admin/config.yml]: :(

**Pull request merged**